content/applications/inventory_and_mrp/repairs/repair_orders.rst

```diff
@@ -0,0 +1,258 @@
+Doing so opens the delivery order (DO) form.
+
+From this form, click :guilabel:`Return`. This opens a :guilabel:`Reverse Transfer` pop-up window.
+
+.. image:: repair_orders/repair-orders-reverse-transfer.png
+   :align: center
+   :alt: Reverse transfer pop-up window on delivery order form.
+
+This pop-up lists the :guilabel:`Product` included in the order, the :guilabel:`Quantity` delivered
+to the customer, and the :guilabel:`Unit of Measure` the product was in.
+
+Click the value in the :guilabel:`Quantity` field to change the quantity of the product to be
+returned, if necessary.
+
+Click the :guilabel:`🗑️ (trash)` icon at the far-right of the product line to remove it from the
+return, if necessary.
+
+Once ready, click :guilabel:`Return` to confirm the return. This creates a new receipt for the
+returned products.
+
+Once the product has been returned to the warehouse, receipt of the return can be registered in the
+database by clicking :guilabel:`Validate` from the reverse transfer form.
+
+.. tip::
+   Once a reverse transfer for a return is validated, the value in the :guilabel:`Delivered` column
+   on the original |SO| updates to reflect the difference between the original :guilabel:`Quantity`
+   ordered, and the :guilabel:`Quantity` returned by the customer.
+
+   .. image:: repair_orders/repair-orders-quantity-delivered.png
+      :align: center
+      :alt: Delivered and Quantity columns on sales order after return.
+
+.. _repairs/repair_orders/repair:
+
+Create repair order
+===================
+
+Once products have been returned, their repairs can be tracked by creating a repair order (RO).
+
+To create a new |RO|, navigate to :menuselection:`Repairs app`, and click :guilabel:`New`. This
+opens a blank |RO| form.
+
+.. image:: repair_orders/repair-orders-left-hand-form.png
+   :align: center
+   :alt: Left-hand side of blank repair order form.
+
+On this form, begin by selecting a :guilabel:`Customer`. The customer selected should be for whom
+the order will be invoiced and delivered.
+
+In the :guilabel:`Product to Repair` field, click the drop-down menu to select the product that
+needs repair. If necessary, click :guilabel:`Search More...` to open a :guilabel:`Search: Product to
+Repair` pop-up window, and browse all products in the database.
+
+Once a :guilabel:`Product to Repair` is selected, a new :guilabel:`Product Quantity` field appears
+below it. In that field, enter the quantity (in a `0.00` format) of the product that requires
+repair.
+
+To the right of that value, click the drop-down list to select the unit of measure (UoM) for the
+product.
+
+In the :guilabel:`Return` field, click the drop-down menu and select the return order from which the
+product to be repaired comes from.
+
+In the :guilabel:`Under Warranty` field, tick the checkbox if the product being repaired is covered
+by a warranty. If ticked, the sales price is set to `0` for all products transferred from the repair
+order.
```

Comment:
Totally up to you how you'd like to describe this, but here's a suggestion!
```suggestion
Tick the :guilabel:`Under Warranty` checkbox, if the product being repaired is covered
by a warranty. If ticked, the :guilabel:`Customer` is not charged for all the parts used in the repair
order.
```

Is this still correct? I wanted to clarify your original sentence:
> If ticked, the sales price is set to `0` for all products transferred from the repair
order.

Reply:
Suggestion implemented ✅